mackup/applications/wtfutil.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = wtf
+
+[configuration_files]
+~/.config/wtf/config.yml
```

Comment:
This can't work. Did you try it?
See https://github.com/lra/mackup/tree/master/doc#add-support-for-an-application-using-the-xdg-directory